libtailscale/interfaces.go

```diff
@@ -77,10 +77,16 @@ type VPNServiceBuilder interface {
 	AddDNSServer(string) error
 	AddSearchDomain(string) error
 	AddRoute(string, int32) error
+	ExcludeRoute(string, int32) error
 	AddAddress(string, int32) error
 	Establish() (ParcelFileDescriptor, error)
 }
 
+// IpPrefix corresponds to Android's IpPrefix
+type IpPrefix interface {
```

Comment:
IIUC, this doesn't need to be defined here because we don't use it anywhere on the Go side.

Reply:
IPPrefix in Go per https://go.dev/wiki/CodeReviewComments#initialisms